proxysql-admin

```diff
@@ -2002,6 +2032,9 @@ function syncusers() {
               ('${user}', '${password}', 1, $WRITER_HOSTGROUP_ID)"
           check_cmd $? "$LINENO" "Failed to add the user ($user) from PXC to ProxySQL database."\
                                "\n-- Please check the ProxySQL connection parameters and status."
+          if [[ $ADD_QUERY_RULE -eq 1 ]];then
```

Comment:
Since we're adding the query rules when we add a user, do we want to remove the query rules when removing a user?

Reply:
Added delete query rule portion in syncusers method 4ef41369f37bd111a34a5b6ebc8df25cbab7ea21